uaclient/config.py

```diff
@@ -526,6 +526,16 @@ def status(self, show_beta=False) -> "Dict[str, Any]":
 
         Write the status-cache when called by root.
         """
+
+        # Try to remove fix reboot notices if not applicable
+        if not util.should_reboot():
```

Comment:
Should we also check presence of os.path.exists(self.data_path("marker-reboot-cmds") too before clearing the reboot notices?
```suggestion
        if not util.should_reboot() and not os.path.exists(self.data_path("marker-reboot-cmds") :
```

Reply:
We could, but since I am only removing `fix` related notices, I don't think it is necessary. The other notices are handled in different situations, like those FIPS ones:
https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/entitlements/fips.py#L187